### ISSUE TITLE
Performance optimization for populate_history

### ISSUE THREAD

**Issue by vdboor** [treyhunner/django-simple-history#423]
**Problem Statement**
Our database had more then 500K of product rows, and more then 1M orders.
Running `manage.py populate_history` had such performance impact that we had to abort this. It took quite some time getting the history tables filled with an initial version.

**Describe the solution you'd like**
In the end, we've monkey-patched the `bulk_history_create()` function to perform a raw SQL `INSERT INTO .. SELECT FROM` query. This gives a tremendous speedup:

```
def bulk_history_create(model, history_model, batch_size):
    history_fields = sorted(get_model_columns(model) & get_model_columns(history_model))
    sql_fields = '"{0}"'.format('", "'.join(history_fields))

    with connection.cursor() as cursor:
        query = """
            INSERT INTO "{history_table}" (
                {sql_fields}, "history_date", "history_type"
            )
            SELECT {sql_fields}, %s, %s FROM "{model_table}"
        """.format(
            history_table=history_model._meta.db_table,
            model_table=model._meta.db_table,
            sql_fields=sql_fields
        )
        cursor.execute(query, params=[now(), TYPE_ADD])
```

I'm not sure this SQL statement works across all possible engines, or that it could only be used in case of PostgreSQL for example. I do hope this code helps to add this performance improvement for the databases that support this.

**Comment by rossmechanic** [comment]
What version of django-simple-history are you using?

**Comment by vdboor** [comment]
When we worked on this, it was the 1.9.0 release.
Currently we're also on the latest 2.3.0 release.

I noticed you've changed the import a lot already 👍 I still think this direct SQL approach gives a better performance, but I haven't tested the different with the latest release yet.

**Comment by rossmechanic** [comment]
I think the updated version is performant, but if I get complaints otherwise, I'm happy to look into it. I think the version available in 2.3.0 is more generalizable and extensible. Going to close this issue.
